### ISSUE TITLE
Accessibility 

### ISSUE THREAD

**Issue by Chenerywoman** [TR-SUNSHINE/Frontend#2]
@thurschm2 just watching the accessibility video - I agree it is really inspiring and informative :-):

- where he talks about not asking for unnecessary information.   Do we actually need first name & surname?  Would username & password be enough?  Do we need email?  Perhaps in a commercial site, but are we planning to use email at all?  I don't know the answer, but just wanted to flag it up. 

- should we put "or" between 'login' and 'register' on the welcome page?  Or is that insulting users' intelligence?! 

- descriptive alt tags or empty - really interesting point - we will need to check these

- "anything below 18 point font becomes harder to read" - just checked: A point (pt) is equal to 0.352778 millimeters, 0.0138889 inches, or **1.333 pixels.** ...so, we need to cross check against our font sizes.  

- we can use Chrome Dev tools to check contrast

**Comment by thurschm2** [comment]
- re unnecessary info: a user remembers firstname & surname better than username, we need email to acertain uniqueness of user. this raises the issue of user uniqueness based on surname+firstaname+email. can we therefore have 2 user with same email???

- i suggest: Login --- or ----- Register
- - we need to do a contrast check across all pages once created, but our colour scheme has a good rate for that

**Comment by devagx** [comment]
I think the application should check a user exists or not based on just email address and not using their name. But I think we should include firstname and surname on the register page still. We may want to use that information now or in the future to personalise their pages.

**Comment by Chenerywoman** [comment]
Summary of my comments on other (closed) accessibility issue.  
Bootstrap 'primary' button fails accessibility on contrast of blue colour against white.
Changed to a darker blue in our colour scheme which passes.
Text size of button can be changed so that it is a minimum of 16px, which I have read is the minimum (although guidance does not state specifically.) Guidance from Blue Prism was 24px but that looks too big for the page.

**Comment by Chenerywoman** [comment]
@thurschm2 Yes, I like the idea of personalising the pages, e.g. "Myriam's Sunshine Today...".   So, first name, surname, email, postcode (if using for location data).
